awxkit/awxkit/cli/resource.py

```diff
@@ -123,6 +126,51 @@ def handle(self, client, parser):
         }
 
 
+class Import(CustomCommand):
+    name = 'import'
+    help_text = 'import resources into Tower'
+
+    def handle(self, client, parser):
+        if client.help:
+            parser.print_help()
+            raise SystemExit()
+
+        data = json.load(sys.stdin)
```

Comment:
First time user interface feedback. The first time I ran `awx import` I suspected my awx server was going very slow. `awx` client was actually just waiting for stdin. That was unexpected. Maybe expect a filename by default and allow users to specify `--stdin` as the non-default.

Reply:
Similarly, the default for `export` should be an expected filename and `--stdout` as a non-default. In this case, it took a while to run through my AWX instance exporting. The output was then printed to STDOUT and I regretted not adding a `> /tmp/export.json`

Reply:
I agree, file-based I/O ought to be available.  Output has implications for the other subcommands, though, since I'm not doing anything specific for that.

Reply:
Maybe check `sys.stdout.isatty()` early in the process and print a warning and exit if it's true.  And if you really want, have another command line flag that suppresses the warning and allows you to send the output to your tty if that's what you really want to do.  And the converse for stdin.